netbox/ipam/utils.py

```diff
@@ -4,20 +4,30 @@
 from .models import Prefix, VLAN
 
 
-def add_available_prefixes(parent, prefix_list):
+def add_requested_prefixes(parent, prefix_list, request):
```

Comment:
My only nit here is that we're passing the whole request object. This should instead be two optional kwargs (e.g. `show_available` and `show_assigned`) which default to True. This will ensure that the function can be used outside the context of a specific request if needed.

Reply:
I'm happy to tackle this myself since this has already been approved.